frag-nav/build.gradle

```diff
@@ -69,13 +69,13 @@ android {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
-    implementation "com.android.support:support-fragment:$rootProject.ext.supportVersion"
-    implementation "com.android.support:support-annotations:$rootProject.ext.supportVersion"
+    implementation "androidx.fragment:fragment:$rootProject.ext.androidxVersion"
+    implementation "androidx.annotation:annotation:$rootProject.ext.androidxVersion"
 
     testImplementation "junit:junit:$rootProject.ext.junitVersion"
     testImplementation "org.mockito:mockito-core:2.13.0"
     testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.0.0-alpha03"
-    testImplementation "org.robolectric:robolectric:3.7"
+    testImplementation 'org.robolectric:robolectric:4.0-alpha-3-SNAPSHOT'
```

Comment:
Had to upgrade to 4.0 because 3.x didn't support API 28

Reply:
4.1.0 stable is available now